js/app.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+var delay;
+var editor = CodeMirror.fromTextArea(document.getElementById(''), {
```

Comment:
I'm not familiar with CodeMirror, are you missing identifying the element id here?